platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/maps/widgets/AttributionTest.java

```diff
@@ -125,6 +125,7 @@ public void testImproveMapLink() {
   }
 
   @Test
+  @Ignore
```

Comment:
I understand this fails on older devices? How about adding something like [this](https://stackoverflow.com/a/53647793/9126211):
```
assumeTrue(
       "Can only run on API Level X or newer because of reasons",
       Build.VERSION.SDK_INT >= X
   )
```